test/platform/vsts/vsts-helper.spec.js

```diff
@@ -298,4 +298,26 @@ describe('platform/vsts/helpers', () => {
       expect(res).toMatchSnapshot();
     });
   });
+
+  describe('getProjectAndRepo', () => {
+    it('should return the object with same strings', async () => {
```

Comment:
If this is jasmine (I was lazy and didn't look), then you can speed up your tests by a significant amount by using the done callback inside async tests. 

```js
it('test', async (done) => {
  try {
    await something();
    done();
  } catch (error) {
    done.fail(error);
  }
});
```

Reply:
we use jest

Reply:
The same applies to jest as show [here](https://facebook.github.io/jest/docs/en/asynchronous.html)

Reply:
Ah, got it.